cli/support.go

```diff
@@ -48,14 +51,52 @@ func (r *RootCmd) supportBundle() *serpent.Command {
 			r.InitClient(client),
 		),
 		Handler: func(inv *serpent.Invocation) error {
-			var (
-				log = slog.Make(sloghuman.Sink(inv.Stderr)).
-					Leveled(slog.LevelDebug)
-				deps = support.Deps{
-					Client: client,
-					Log:    log,
-				}
+			var cliLogBuf bytes.Buffer
+			cliLogW := sloghuman.Sink(&cliLogBuf)
+			cliLog := slog.Make(sloghuman.Sink(inv.Stderr), cliLogW)
+			if r.verbose {
+				cliLog = cliLog.Leveled(slog.LevelDebug)
+			}
+
+			vi := defaultVersionInfo()
+			cliLog.Info(inv.Context(), "version info",
+				slog.F("version", vi.Version),
+				slog.F("build_time", vi.BuildTime),
+				slog.F("external_url", vi.ExternalURL),
+				slog.F("slim", vi.Slim),
+				slog.F("agpl", vi.AGPL),
+				slog.F("boring_crypto", vi.BoringCrypto),
 			)
+			cliLog.Info(inv.Context(), "invocation", slog.F("args", strings.Join(os.Args, " ")))
+
+			if !confirm {
+				ans, err := cliui.Prompt(inv, cliui.PromptOptions{
+					Text:      cliui.Bold("Note: ") + cliui.Wrap("While we try to sanitize sensitive data from support bundles, we cannot guarantee that they do not contain information that you or your organization may consider sensitive.\n") + cliui.Bold("Please confirm that you will:\n") + "  - Review the support bundle before distribution\n  - Only distribute it via trusted channels.\n",
+					Default:   cliui.ConfirmNo,
+					Secret:    false,
+					IsConfirm: true,
+				})
+				if err != nil || ans != cliui.ConfirmYes {
+					return err
+				}
+				cliLog.Info(inv.Context(), "user confirmed manually", slog.F("answer", ans))
+			} else {
+				cliLog.Info(inv.Context(), "user auto-confirmed")
```

Comment:
nit: debug or trace level?

Reply:
Yeah fair, we don't want unnecessary output.